contracts/stellar-interchain-token-service/src/flow_limit.rs

```diff
@@ -56,18 +56,24 @@ impl FlowDirection {
 
         ensure!(flow_amount <= flow_limit, ContractError::FlowLimitExceeded);
 
-        let new_flow = self
-            .flow(env, token_id.clone())
-            .checked_add(flow_amount)
+        let flow_amount_ = flow_amount as u128;
+        let flow_limit_ = flow_limit as u128;
```

Comment:
```suggestion
        let flow_amount = flow_amount as u128;
        let flow_limit = flow_limit as u128;
```
shadowing the previous names is more idiomatic in rust